projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -148,20 +150,24 @@ export class GoTableComponent implements OnInit, OnChanges, AfterViewInit {
   }
 
   nextPage(): void {
-    this.localTableConfig.pageConfig.offset = this.localTableConfig.pageConfig.offset + this.localTableConfig.pageConfig.perPage;
+    this.setPage(this.localTableConfig.pageConfig.offset + this.localTableConfig.pageConfig.perPage);
 
     this.tableChangeOutcome();
   }
 
-  isLastPage(): boolean {
+  isLastPage(offset: number): boolean {
     const { pageConfig, tableData, totalCount }:
     {
       pageConfig: GoTablePageConfig,
       tableData: any[],
       totalCount: number
     } = this.localTableConfig;
 
-    return ((pageConfig.offset + pageConfig.perPage) >= tableData.length) && ((pageConfig.offset + pageConfig.perPage) >= totalCount);
+    if (offset === undefined) {
```

Comment:
Would `offset` be `undefined` if it's a required param? I could see if you wanted to check for this if it was optional and then doing something with it, but not sure this is needed.

Reply:
Which, it might be easier to set a default for `offset`?
```
isLastPage(offset: number = 0): boolean {

}
```
Then we don't have to pass `0` in all the method calls in the file and it will never be `undefined`. Unless I am missing something.

Reply:
I think you're right here. I think we want it to be a required param